inst/views/components/metrics_charts.html

```diff
@@ -254,6 +257,45 @@
     request.send();
   };
 
+  // determine ticks for x-axis
+  this.$get_x_axis_ticks = function(total_epochs) {
+
+      // TODO: should depend on width of chart?
+      var total_ticks = 10;
```

Comment:
My approach was to provide manual ticks for epochs/steps <= 30 as I felt like no matter the width of the display 30 ticks worked pretty well. So I think you could define a static `total_ticks` of anywhere up to 30 and be okay.

Reply:
I went with 10 here as I felt 30 can get pretty crowded in the Viewer pane (especially if the RStudio window is small, and there are a large number of steps [axis ticks with 3 or 4 digits])